resolver/resolver.go

```diff
@@ -84,33 +259,34 @@ func (resolver *Resolver) QueryState(ctx context.Context, id interface{}, params
 }
 
 func (resolver *Resolver) QueryTxs(ctx context.Context, id interface{}, params *jsonrpc.ParamsQueryTxs, req *http.Request) jsonrpc.Response {
-	var offset int
-	if params.Offset == nil {
-		// If the offset is nil, set it to 0.
-		offset = 0
-	} else {
-		offset = int(*params.Offset)
-	}
+	return resolver.handleMessage(ctx, id, jsonrpc.MethodQueryTxs, *params, req, false)
+	// var offset int
```

Comment:
Why is this code commented out?

Reply:
Ah sorry, that was by mistake; will revert